packages/coverage-c8/package.json

```diff
@@ -41,14 +41,17 @@
     "dev": "rollup -c --watch --watch.include 'src/**'",
     "prepublishOnly": "pnpm build"
   },
+  "peerDependencies": {
+    "vitest": ">=0.28.0 <1"
```

Comment:
This means that coverage packages set minimum requirement of current "breaking change" release, and upper limit for v1. This may still cause some incompatibility issues if API is broken from the `vitest` side.

On the other hand, if we use just `"^0.28.0"` we need to bump this version on every minor/breaking release. `0.29.0` does not match `"^0.28.0"`.

Any thoughts, which one should we end up using?

Reply:
I think >= is better than using a caret here, because it doesn’t match minor versions, if major is 0. When we release v1, we can update this to use a caret

Reply:
Yes, that sounds good. And now when ever we need to require newer version as peer dependency, we'll just manually bump the version here. For example, after merging #2786 this sound be changed to `">=0.29.0 <1"` before publishing new minor release. 
After v1 this will get easier.